app/src/main/java/org/sopt/do_sopt_compose/ui/components/MainButton.kt

```diff
@@ -0,0 +1,38 @@
+package org.sopt.do_sopt_compose.ui.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun MainButton(
+    onClick: () -> Unit,
+    text: String,
+    modifier: Modifier = Modifier,
+) {
+    Button(
+        onClick = onClick,
+        content = {
+            Text(text = text)
+        },
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(8.dp)
+            .clickable { onClick() },
+    )
```

Comment:
이거 약간 아래처럼 하는 걸 많이 본 거 같은데 이릏게 바꿔보심 어떻까요?
```suggestion
    Button(
        onClick = onClick,
        modifier = modifier
            .fillMaxWidth()
            .padding(8.dp)
            .clickable { onClick() },
    ) {
           Text(text = text)
      }
```

Reply:
그리고 onclick이랑 .clickable 중에 하나만 있어도 되지.. 않을까요...? 둘 다 만드신 의도가 궁금합니다!!